_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -0,0 +1,84 @@
+The way enmeshed handles updates to your personal data is by succeeding old [Attributes]({% link _docs_integrate/data-model-overview.md %}#attributes) with new ones.
+Instead of simply replacing old data, a completely new Attribute is created which succeeds its predecessor.
+Thus, you and your peers are provided a coherent history of all past versions.
+How the Attribute succession works in detail depends on the type of Attribute.
+
+## Succeeding own IdentityAttributes
+
+When talking about [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute), we need to distinguish three cases:
+
+- The Identity maintains an unshared Attribute about itself. This IdentityAttribute is stored in the `content` field of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute), whose `shareInfo` is undefined. Since this LocalAttribute is created for the Identity's private repository of Attributes, it is referred to as **RepositoryAttribute**.
+- When sharing a RepositoryAttribute with a peer, a copy of the IdentityAttribute is created for the Sender and stored in the `content` field of a LocalAttribute with a defined `shareInfo`. We call this LocalAttribute an **own shared IdentityAttribute**.
+- Receiving a shared IdentityAttribute from a peer leads to the creation of a LocalAttribute with according `content` and a defined `shareInfo` for the Recipient. We call this LocalAttribute a **peer shared IdentityAttribute**.
+
```

Comment:
Doppelt gemoppelt? Steht oben schon.

Reply:
could we switch back to english in PRs? :)